.github/workflows/integration.yml

```
@@ -10,6 +10,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: oven-sh/setup-bun@v2
       - run: bun install
+      - run: bun run prettier
```

도구는 언제든지 바뀔 수 있는 부분이라 일반적인 커멘드 명을 추천드립니다.

```suggestion
      - run: bun run format
```